packages/ember-glimmer/lib/syntax/outlet.js

```diff
@@ -299,9 +264,12 @@ class TopLevelOutletLayoutCompiler {
 
 TopLevelOutletLayoutCompiler.id = 'top-level-outlet';
 
-class OutletComponentDefinition extends AbstractOutletComponentDefinition {
+class OutletComponentDefinition extends ComponentDefinition {
   constructor(outletName, template) {
-    super(MANAGER, outletName, template);
+    super('outlet', MANAGER, null);
+    this.outletName = outletName;
+    this.template = template;
+    generateGuid(this);
```

Comment:
It was previously done in the `AbstractOutletComponentDefinition` which is removed (since it does very little for us). I just kept it from there when I refactored out the abstract parent.